examples/bpi-monolith-test-suite/bpi/components/zkcircuits/json/orderCircuitjson.go

```diff
@@ -0,0 +1,19 @@
+package json
+
+type VerifyInput struct {
+	I OrderCircuitInput `json:"I"`
```

Comment:
this is a bit confusing to me, when you verify input should you only pass public input + proof? but here you can pass both public and private inputs?

Reply:
Correct this should only be using the public witness not the full witness. Proving requires the full witness and verifying requires only the public witness.